src/Expectation.php

```diff
@@ -715,6 +715,33 @@ public function toMatchConstraint(Constraint $constraint): Expectation
         return $this;
     }
 
+    /**
+     * Asserts that executing value throws an exception.
+     *
+     * @param string|\Closure $exception string: the exception class
```

Comment:
If the given string, is not a class, can you assert that an exception as thrown with the given message?